testing.rst

```diff
@@ -801,6 +801,12 @@ their type::
     $form['my_form[field][O]']->upload('/path/to/lucas.jpg');
     $form['my_form[field][1]']->upload('/path/to/lisa.jpg');
 
+.. tip::
+
+    All field names in forms generated by Symfony are prefixed with the form's name.
+    If the name is autogenerated you can get it using
```

Comment:
This sounds misleading IMO. For the method it doesn't really matter how the name originates. Maybe we should reword it to something like this:

> Use the `getName()` method if you need to access the name of a form.

Reply:
Name of the field is not that obvious on the first sight, so I believe a hint is needed to make it clear. Also, using `getName` method in tests can make them a little less fragile.